x-pack/solutions/security/plugins/security_solution/public/entity_analytics/components/privileged_user_monitoring/components/privileged_access_detection/index.tsx

```diff
@@ -0,0 +1,118 @@
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { EuiEmptyPrompt, EuiFlexGroup, EuiPanel, EuiProgress } from '@elastic/eui';
+import React from 'react';
+import { useMutation, useQuery } from '@tanstack/react-query';
+import { PrivilegedAccessDetectionMLPopover } from './pad_ml_popover';
+import { HeaderSection } from '../../../../../common/components/header_section';
+import { PRIVILEGED_USER_ACTIVITY_QUERY_ID } from '../privileged_user_activity/constants';
+import { useQueryToggle } from '../../../../../common/containers/query_toggle';
+import { PrivilegedAccessDetectionChart } from './pad_chart';
+import { usePrivilegedAccessDetectionRoutes } from './pad_routes';
+import { PrivilegedAccessDetectionInstallPrompt } from './pad_install_prompt';
+import { PrivilegedAccessDetectionViewAllAnomaliesButton } from './pad_view_all_anomalies_button';
+
+const TITLE = i18n.translate(
+  'xpack.securitySolution.entityAnalytics.privilegedUserMonitoring.topPrivilegedAccessDetectionAnomalies.title',
+  { defaultMessage: 'Top privileged access detection anomalies' }
+);
+
+const PRIVILEGED_ACCESS_DETECTIONS_QUERY_ID = 'privileged-access-detection-query';
+
+export const PrivilegedAccessDetectionsPanel: React.FC<{ spaceId: string }> = ({ spaceId }) => {
+  const { toggleStatus, setToggleStatus } = useQueryToggle(PRIVILEGED_USER_ACTIVITY_QUERY_ID);
+
+  const {
+    getPrivilegedAccessDetectionStatus,
+    setupPrivilegedAccessDetectionMlModule,
+    installPrivilegedAccessDetectionPackage,
+  } = usePrivilegedAccessDetectionRoutes();
+
+  const { data: padInstallationStatus, refetch: refetchInstallationStatus } = useQuery(
+    ['padInstallationStatus'],
+    getPrivilegedAccessDetectionStatus,
+    {
+      refetchInterval: 10000,
+    }
+  );
+
+  const setupMlModuleMutation = useMutation({ mutationFn: setupPrivilegedAccessDetectionMlModule });
+  const installPrivilegedAccessDetectionPackageMutation = useMutation({
+    mutationFn: installPrivilegedAccessDetectionPackage,
+  });
+
+  const currentlyInstalling =
+    installPrivilegedAccessDetectionPackageMutation.isLoading || setupMlModuleMutation.isLoading;
+
+  const install = async () => {
+    await installPrivilegedAccessDetectionPackageMutation.mutateAsync();
+    await setupMlModuleMutation.mutateAsync();
+    await refetchInstallationStatus();
+  };
+
+  const packageInstallationComplete =
+    padInstallationStatus?.packageInstallationStatus === 'complete' &&
+    padInstallationStatus?.mlModuleSetupStatus === 'complete';
+
+  return (
+    <>
+      {!padInstallationStatus && <div css={{ height: '400px' }} />}
+      {padInstallationStatus && !packageInstallationComplete && !currentlyInstalling && (
+        <PrivilegedAccessDetectionInstallPrompt install={install} />
+      )}
+
+      {currentlyInstalling && (
+        <>
+          <EuiEmptyPrompt
+            css={{ minWidth: '100%' }}
```

Comment:
instead of CSS could we achieve this with EUiFlexGroup/EuiFlexItem and grow=true? it might replace all the fragments wrapping these components, not 100% here just an idea to avoid the CSS